LBHackney-IT/lbh-social-care
# Add form components

**What**  
Added basic form components:
- Button
- Checkbox
- Date Input
- Radio
- Select
- Text Input
- Text Area

and some utility components"
- Error Message
- Error Summary

Added Storybook to showcase the components ☝️ 

<img width="747" alt="Screenshot 2020-10-21 at 11 52 24" src="https://user-images.githubusercontent.com/435399/96710654-a3de1a80-139c-11eb-8296-a77980800d72.png">


**How to test it**
- checkout this branch
- install dependencies
- run storybook `yarn storybook`
- check that everything looks nice 🎉
